app/models/good_job/job.rb

```diff
@@ -60,7 +60,7 @@ def table_name=(_value)
 
     scope :unfinished_undiscrete, -> { where(finished_at: nil, retried_good_job_id: nil, is_discrete: [nil, false]) }
 
-    # The job's ActiveJob UUID
+    # The job's Active Job UUID
```

Comment:
This is a UUID that's turned into a string?

Reply:
Yep, that's how Ruby represents UUIDs:

```ruby
❯ bin/rails c
Loading development environment (Rails 7.1.1)
[1] pry(main)> SecureRandom.uuid.class
=> String
```